### PR TITLE
FINERACT-822 enable ImmutableEnumChecker error

### PR DESCRIPTION
**Pull request by percyashu**
FINERACT-822

### CODE REVIEW

File: fineract-provider/build.gradle [hunk 1]
```diff
@@ -436,7 +436,8 @@ modernizer {
 
     ignorePackages = []
     ignoreClassNamePatterns = [
-        '.*AbstractPersistableCustom'
+        '.*AbstractPersistableCustom',
+        '.*EntityTables'
```
Comment (percyashu) [inline fineract-provider/build.gradle:448]: Errorprone wants all fields in Enumeration to Immutable, hence Integer[] is require to be ImmutableList<Integer> and modernizer wants all ImmutableList.of to be List.of which Error prone doesn't consider as Immutable, hence had to add EntityTables to this list.

Comment (vorburger) [reply fineract-provider/build.gradle:448]: understood. Thanks for the explanation.

**Review by vorburger — APPROVED** [review]

**Comment by percyashu** [comment]
@awasum @xurror @vorburger merge this?

**Comment by awasum** [comment]
Squash this? Otherwise looks good.

**Review by awasum — APPROVED** [review]

**Pull request merged**